docker/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "3"
+services:
+  db:
+    image: "postgres:11"
+    container_name: "postgres"
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_DB: cag_db
+      POSTGRES_HOST_AUTH_METHOD: trust
+    volumes:
+      - /var/lib/postgresql/data
```

Comment:
What does this file do? Is it meant to connect something?

Reply:
So this is Docker. We use docker to spin up a containerized postgres database on our local machines quickly and easily. I still have to configure ours correctly, but I'll give a docker tutorial soon! More to come on this